core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,1073 @@
+		<xsl:param name="pipeline"/>
+		<xsl:param name="id"/>
+		<xsl:param name="specialIds"></xsl:param>
+<!--		<xsl:param name="inputValidatorId"/>-->
+<!--		<xsl:param name="outputValidatorId"/>-->
+		<xsl:param name="firstPipe"/>
+
+		<xsl:variable name="internalName">
+			<xsl:text>ReceiveR</xsl:text>
+			<xsl:value-of select="replace(@name, ' ', '__')"/>
+			<xsl:value-of select="$id"/>
+		</xsl:variable>
+
+<!--		<firstPipe><xsl:value-of select="$firstPipe"/></firstPipe>-->
+<!--		<specialIdsTest1><xsl:copy-of select="$specialIds"/></specialIdsTest1>-->
+<!--		<specialIdsTest2><xsl:copy-of select="$specialIds/*"/></specialIdsTest2>-->
+<!--		<specialIdsTest3><xsl:value-of select="$specialIds/inputValidator/text()"/></specialIdsTest3>-->
+<!--		<concatTest><xsl:value-of select="concat('inputValidator', $specialIds/inputValidator/text())"/></concatTest>-->
+		<element>
+			<identification>
+				<xsl:value-of select="concat($id, '|receiver|', $internalName)"/>
+			</identification>
+			<value>
+				<xsl:text>	</xsl:text>
+				<xsl:value-of select="$internalName"/>
+				<xsl:text>{{Receiver</xsl:text>
+				<xsl:text disable-output-escaping="yes">&amp;lt;br></xsl:text>
+				<xsl:value-of select="@name"/>
+				<xsl:text>}}</xsl:text>
+				<xsl:text>&#10;</xsl:text>
+			</value>
+		</element>
+
+		<xsl:variable name="newFlowHistory">
+			<pipe>
+				<xsl:value-of select="$internalName"/>
+			</pipe>
+		</xsl:variable>
+
+		<!--This is where we actually forward to where we want to go. We don't use the forward template for this,
+		 because we may have to forward to the pipelines inputvalidator, which the forward template can't handle. Maybe add this later, like for the outputValidator-->
+		<xsl:choose>
+			<!--If the firstPipe param is the same as the inputValidatorId concatenated after 'inputValidator', then we know we have to start with the inputValidator-->
+			<xsl:when test="$firstPipe = concat('inputValidator', $specialIds/inputValidator/text())">
+				<xsl:call-template name="forward">
+					<xsl:with-param name="pipeline" select="$pipeline"/>
+					<xsl:with-param name="id" select="$id"/>
+					<xsl:with-param name="specialIds" select="$specialIds"/>
+					<xsl:with-param name="flowHistory" select="$newFlowHistory"/>
+					<xsl:with-param name="origin" select="concat('ReceiveR', @name)"/>
+					<xsl:with-param name="text" select="' '"/>
+					<xsl:with-param name="destination" select="concat('inputValidator', $specialIds/inputValidator/text())"/>
+					<xsl:with-param name="isValidationFlow" select="false()"/>
+					<xsl:with-param name="specialForward" select="'inputValidator'"/>
+					<xsl:with-param name="specialForwardValue"><!--This param is used to let the inputvalidator know where the success (or other) forwards go to when it doesn't defined them its self-->
+							<forward name="success">
+								<xsl:attribute name="path">
+									<xsl:choose>
+										<xsl:when test="$pipeline//pipeline/@firstPipe">
```

Comment:
// zorgt ervoor dat je de hele XML $pipeline door gaat zoeken naar pipeline (en op andere plekken waar je // hebt vergelijkbaar), ik zou verwachten dat /pipeline het ook zou doen

Reply:
dit is niet meer van toepassing. Nu wordt '//' alleen nog gebruikt voor het vinden van exit elementen die mogelijk in een exits element zitten.